apps/ledger/src/credential-definition/repositories/credential-definition.repository.ts

```diff
@@ -5,6 +5,8 @@ import { credential_definition, org_agents, org_agents_type, organisation, schem
 import { Injectable, Logger } from '@nestjs/common';
 import { ResponseMessages } from '@credebl/common/response-messages';
 import { BulkCredDefSchema, CredDefSchema } from '../interfaces/credential-definition.interface';
+import { ICredDefData } from '@credebl/common/interfaces/cred-def.interface';
+import { SortValue, sortValue } from 'apps/api-gateway/src/enum';
```

Comment:
Please create this `enum` in libs folder

Reply:
Resolved